packages/datepickers/src/elements/DatepickerRange/utils/useDatepickerRangeContext.ts

```diff
@@ -17,7 +17,6 @@ export interface IDatepickerRangeContext {
   maxValue?: Date;
   startValue?: Date;
   endValue?: Date;
-  onChange?: (values: { startValue?: Date; endValue?: Date }) => void;
```

Comment:
Since `onChange` is removed from the context, should it also be removed from being [passed down](https://github.com/zendeskgarden/react-components/blob/main/packages/datepickers/src/elements/DatepickerRange/DatepickerRange.tsx#L146) the `<DatepickerRangeContext.Provider>`?

Reply:
Yeah. Might need to go in the opposite direction with this one and actually apply [documented](https://github.com/zendeskgarden/react-components/blob/77499ab31d87fb7a012c8c6746e64e41b2591b08/packages/datepickers/src/elements/DatepickerRange/DatepickerRange.tsx#L45-L52) `onChange` to the intended target. 🕵️